homepage/homepage_web_client.go

```diff
@@ -11,47 +11,45 @@ import (
 	"github.com/ONSdigital/log.go/v2/log"
 )
 
-type HomepageWebClient struct {
-	HomepageUpdater
+type WebClient struct {
+	Updater
 	cache           *cache.HomepageCache
 	navigationCache *cache.NavigationCache
 	languages       []string
 }
 
-func NewHomePageWebClient(ctx context.Context, clients *Clients, updateInterval time.Duration, languages []string) (HomepageClienter, error) {
+func NewHomePageWebClient(ctx context.Context, clients *Clients, updateInterval time.Duration, languages []string) (Clienter, error) {
```

Comment:
Name should change now we've renamed `HomepageWebClient` to just `WebClient`
```suggestion
func NewWebClient(ctx context.Context, clients *Clients, updateInterval time.Duration, languages []string) (Clienter, error) {
```